src/status_im/contexts/wallet/common/token_value/view.cljs

```diff
@@ -3,50 +3,63 @@
             [utils.i18n :as i18n]
             [utils.re-frame :as rf]))
 
+(defn watch-only-token-value-drawer
+  []
+  [quo/action-drawer
+   [[{:icon                :i/settings
```

Comment:
I think we want to stop adding features with "to be done".
They can be added but should be under a common feature flag/ a particular feature flag.
Cc @seanstrom @cammellos

Reply:
I would agree, yes, basically it should make sense to the user without being disruptive, so 99.99% of the times, a user would find confusing clicking on something and seeing "Not implemented" (and rather disappointing), with the only exception stuff like an entry "New feature (coming soon :) )" (i.e a teaser), but that's not the case here

Reply:
so hide everything (if you want to keep in development is up to us) behind a feature flag

Reply:
Done